src/main/java/com/google/sps/utils/data/DataCollection.java

```diff
@@ -134,6 +142,89 @@ private static void parseSpeciesTable(String url) throws IOException {
         }
     }
 
+    /**
+    * Scrapes EXTINCT_URL to get a list of all the URLs with information about species
+    * Page Structure:
+    *   <ul>
+    *     <li><a href="LINK"></a>[Text]</li>
+    *     <li><a href="LINK"></a>[Text]</li>
+    *   </ul>
+    *
+    * @return List of all the URLs to go to
+    */
+    private static List<String> parseExtinctList() throws IOException {
+        Document doc = Jsoup.connect(EXTINCT_URL).get();
+        Elements content = doc.getElementsByClass(LIST_CONTENT_CLASS);
+        Elements listItems = content.select("ul > li"); // get list items that are in a nested ul
+        List<String> urls = new ArrayList<String>();
+
+        boolean reachedSpecies = false;
+        for (Element listItem: listItems) {
+            if (listItem.text().contains("Alagoas curassow")) {
+                reachedSpecies = true;
+            }
+
+            // Ignore other list elements
+            if (!reachedSpecies) {
+                continue;
+            }
```

Comment:
Is this meant to be for the end of the list?

Reply:
That's for any bullets before the start of the extinct animals (there were multiple things that matched the "ul > li" format)